[BUG] pax-utils fails to cross compile for RPi aarch64 on gentoo linux

### Describe the bug
It seems the system provided libcap at /lib/libcap.so is found and in return tried to be linked.

### How to reproduce
PROJECT=RPi ARCH=aarch64 DEVICE=RPi4 ./scripts/build pax-utils:host

results in:
```
Host machine cpu family: x86_64
Host machine cpu: x86_64
Dependency libcap found: YES 2.69 (cached)
Found git repository at /android/src/LibreELEC.tv
Has header "endian.h" : YES (cached)
Has header "byteswap.h" : YES (cached)
Has header "sys/endian.h" : NO (cached)
Has header "sys/isa_defs.h" : NO (cached)
Has header "machine/endian.h" : NO (cached)
Has header "linux/seccomp.h" : YES (cached)
Has header "linux/securebits.h" : YES (cached)
Has header "sys/prctl.h" : YES (cached)
Has header "alloca.h" : YES (cached)
Has header "elf-hints.h" : NO (cached)
Has header "glob.h" : YES (cached)
Configuring probes.h using configuration
Fetching value of define "__svr4__" : (undefined) (cached)
Program xmlto found: YES (/usr/bin/xmlto)
Configuring pax-utils.docbook using configuration
../meson.build:158: WARNING: use_fuzzing requires Clang, due to LibFuzzer. Not building fuzzers
Build targets in project: 7

pax-utils 1.3.7

  User defined options
    Native files : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/build/pax-utils-1.3.7/.x86_64-pc-linux-gnu/meson.conf
    bindir       : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/bin
    buildtype    : plain
    libdir       : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/lib
    libexecdir   : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/lib
    localstatedir: /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/var
    prefix       : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain
    sbindir      : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/sbin
    sysconfdir   : /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/etc

Found ninja-1.11.1 at /android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/bin/ninja
Cleaning... 0 files.
[2/5] Linking target scanmacho
FAILED: scanmacho 
/android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/bin/host-gcc  -o scanmacho scanmacho.p/paxmacho.c.o scanmacho.p/scanmacho.c.o -L/android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/lib -Wl,--as-needed -Wl,--no-undefined -Wl,-rpath,/android/src/LibreELEC.tv/build.LibreELEC-RPi4.aarch64-12.0-devel/toolchain/lib -Wl,-rpath,//lib -Wl,-rpath-link,//lib -Wl,--start-group libcommon.a //lib/libcap.so -Wl,--end-group
/usr/lib/gcc/x86_64-pc-linux-gnu/13/../../../../x86_64-pc-linux-gnu/bin/ld: //lib/libcap.so: error adding symbols: file in wrong format
collect2: error: ld returned 1 exit status
```

### Information
 - LibreELEC Version: master 96b4665e4bb8bc16d09bc1d0f1fa7cdd2eeee944
 - Hardware Platform: x86_64 to cross compile for RPi4

--- a/packages/devel/pax-utils/package.mk
+++ b/packages/devel/pax-utils/package.mk
@@ -9,3 +9,5 @@ PKG_SITE="https://wiki.gentoo.org/wiki/Hardened/PaX_Utilities"
 PKG_URL="https://gitweb.gentoo.org/proj/pax-utils.git/snapshot/pax-utils-${PKG_VERSION}.tar.bz2"
 PKG_DEPENDS_HOST="toolchain:host"
 PKG_LONGDESC="ELF utils that can check files for security relevant properties"
+
+PKG_MESON_OPTS_HOST="-Duse_libcap=disabled"

